projects/1.19/assets/the-masquerade-illager-boss/masquerader_mod/lang/zh_cn.json

```
@@ -0,0 +1,37 @@
+{
+  "entity.masquerader_mod.masquerader": "假面愚者",
+  "entity.masquerader_mod.masquerader_clone": "假面愚者",
+  "entity.masquerader_mod.masked_vindicator": "卫道士",
+  "entity.masquerader_mod.masked_evoker": "唤魔者",
+  "entity.masquerader_mod.masked_illusioner": "幻术师",
+  "entity.masquerader_mod.masked_witch": "女巫",
+  "entity.masquerader_mod.masked_villager": "戴面具的村民",
+
+  "sounds.masquerader_mod.masquerader_change_mask": "假面愚者：准备更换面具",
+  "sounds.masquerader_mod.masquerader_laugh": "假面愚者：大笑",
+  "sounds.masquerader_mod.masquerader_prepare_clone": "假面愚者：准备幻术伏击",
+  "sounds.masquerader_mod.masquerader_death": "假面愚者：死亡",
+  "sounds.masquerader_mod.clone_death": "假面愚者分身：消失",
+
+  "advancements.masquerader_mod.invitation_advancement": "化装舞会！",
+  "advancements.masquerader_mod.invitation_advancement_desc": "受邀参加一场高档社交晚会",
+  "advancements.masquerader_mod.ballroom_advancement": "\"时髦的\"迟到",

```

```suggestion
  "advancements.masquerader_mod.ballroom_advancement": "“时髦的”迟到",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

以及，这里`迟到`（Late）的词性是动词吧？`时髦地`？

---

> 标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。 诸如这类的问题请在您翻译的所有文件中全部查明并修改。
> 
> 以及，这里`迟到`（Late）的词性是动词吧？`时髦地`？

应该是 时髦的迟到 ，如果是地，会用，fashionably，不过确实这个 Fashionable Late 没现成的不错的翻译

---

> 标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。 诸如这类的问题请在您翻译的所有文件中全部查明并修改。
> 
> 以及，这里`迟到`（Late）的词性是动词吧？`时髦地`？

用了那个“翩翩”来迟